Manager/TicketManager.php

```diff
@@ -38,8 +38,9 @@ public function __construct(EntityManagerInterface $manager, string $class, bool
      * @throws \Doctrine\ORM\ORMException
      * @throws \Doctrine\ORM\OptimisticLockException
      */
-    public function createTicket(UserInterface $user, Ticket $ticket){
-        $status = $this->ticketStatusManager->getRepository()->findOneBy(['name' => $this->ticketStatusManager->getDefaultStatus()]);
+    public function createTicket(UserInterface $user, Ticket $ticket)
+    {
+        $status = $this->ticketStatusManager->getDefaultStatus();
```

Comment:
Fix the syntax

Remove the empty line

Remove the phpdoc (@var ...) because it is no longer necessary